### ISSUE TITLE
Can't seem to get it working

### ISSUE THREAD

**Issue by Jamesking56** [leaflet-extras/leaflet-providers#150]
Hello,

I'm trying to use MapBox with Leaflet JS using your this project.

Here is my code:

````js
var map = L.map('map').setView(
    [latitude, longitude],
    13
);

L.tileLayer.provider('MapBox.jamesking56.ligeph22').addTo(map);
L.geoJson(geoJson, {}).addTo(map);
````

What am I doing wrong? There's no console errors but I just get a blank grey map with my GeoJSON on it (no tiles)

**Comment by brunob** [comment]
I get 401 Unauthorized errors when i try to access to your map, are you sure this one public ?

**Comment by jieter** [comment]
Yes, same here: 
http://jsfiddle.net/vf9Lga36/

Console output (chrome):
![2015-03-27-095808_912x145_scrot](https://cloud.githubusercontent.com/assets/1470389/6864901/db2266ec-d467-11e4-8175-9ee407bf1510.png)

**Comment by Jamesking56** [comment]
Well it has been set to public on Mapbox...

**Comment by jieter** [comment]
Still get the same `401 (Unauthorized)` messages in the console.

**Comment by jieter** [comment]
@Jamesking56 sorry for not recognizing your problem earlier, should be fixed now by #167.
